src/main/kotlin/org/opensearch/replication/task/index/IndexReplicationTask.kt

```diff
@@ -828,7 +828,11 @@ open class IndexReplicationTask(id: Long, type: String, action: String, descript
             }
         } catch(e: Exception) {
             val err = "Unable to initiate restore call for $followerIndexName from $leaderAlias:${leaderIndex.name}"
+            val aliasErrMsg = "cannot rename index [${leaderIndex.name}] into [$followerIndexName] because of conflict with an alias with the same name"
```

Comment:
This scenario is one of cases in which the replication fails to start and proper message is not thrown. We need to make it generic to cover all cases.

Reply:
+1

Reply:
Yes, For other cases if something fails in restore state then we will get this message in index replication status API "Unable to initiate restore call for $followerIndexName from $leaderAlias:${leaderIndex.name}".

Initially when failed state was not handled properly, we get generic error of "Timed out when waiting for persistent task after 1m". Which might happened because of failure in restore state or error in any other state.

Reply:
Updated the failed state message with whatever exception message will come